<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>354</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qthon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="lw_textures">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 171, 171);</string>
      </property>
      <property name="autoScrollMargin">
       <number>32</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_Selections_As"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Images"/>
    <addaction name="actionExport_Selected_Images"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuHide">
     <property name="title">
      <string>Hide</string>
     </property>
     <addaction name="actionHide_toolbar"/>
     <addaction name="actionHide_statusbar"/>
     <addaction name="actionHide_sidebar"/>
    </widget>
    <widget class="QMenu" name="menuMovable_2">
     <property name="title">
      <string>Movable</string>
     </property>
     <addaction name="actionMovable_toolbar"/>
     <addaction name="actionMovable_sidebar"/>
    </widget>
    <addaction name="menuHide"/>
    <addaction name="menuMovable_2"/>
    <addaction name="separator"/>
    <addaction name="actionView_Detailed"/>
    <addaction name="actionView_Animated"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Sort_Items">
     <property name="title">
      <string>&amp;Sort Items</string>
     </property>
     <addaction name="action_Ascending"/>
     <addaction name="action_Descending"/>
    </widget>
    <widget class="QMenu" name="menuRotate_Textures">
     <property name="title">
      <string>Rotate Te&amp;xtures</string>
     </property>
     <addaction name="actionRotate_Left"/>
     <addaction name="actionRotate_Right"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect_All"/>
    <addaction name="separator"/>
    <addaction name="menu_Sort_Items"/>
    <addaction name="menuRotate_Textures"/>
    <addaction name="separator"/>
    <addaction name="actionDefullbright"/>
    <addaction name="actionRename"/>
    <addaction name="actionResize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="tb_options">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionImport"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <widget class="QToolBar" name="tb_editor">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResize"/>
   <addaction name="actionRename"/>
   <addaction name="actionFlip"/>
   <addaction name="actionMirror"/>
   <addaction name="actionRotate_Left"/>
   <addaction name="actionRotate_Right"/>
   <addaction name="separator"/>
   <addaction name="actionView_Detailed"/>
   <addaction name="actionView_Animated"/>
  </widget>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/document-medium.png</normaloff>:/icons/fugue-icons/document-medium.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>New (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/folder-open.png</normaloff>:/icons/fugue-icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/disk.png</normaloff>:/icons/fugue-icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/disk--pencil.png</normaloff>:/icons/fugue-icons/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save As (Ctrl+Shift+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/minus-button.png</normaloff>:/icons/fugue-icons/minus-button.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/document--arrow.png</normaloff>:/icons/fugue-icons/document--arrow.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="toolTip">
    <string>Import (Ctrl+I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/information-button.png</normaloff>:/icons/fugue-icons/information-button.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/book-question.png</normaloff>:/icons/fugue-icons/book-question.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/arrow-turn-180-left.png</normaloff>:/icons/fugue-icons/arrow-turn-180-left.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo (Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDeselect_All">
   <property name="text">
    <string>Select N&amp;one</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/scissors-blue.png</normaloff>:/icons/fugue-icons/scissors-blue.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="toolTip">
    <string>Cut (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/document-copy.png</normaloff>:/icons/fugue-icons/document-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/clipboard-paste-document-text.png</normaloff>:/icons/fugue-icons/clipboard-paste-document-text.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/image--minus.png</normaloff>:/icons/fugue-icons/image--minus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete (Del)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/document-rename.png</normaloff>:/icons/fugue-icons/document-rename.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;ename</string>
   </property>
   <property name="toolTip">
    <string>Rename (F2)</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/ui-panel-resize.png</normaloff>:/icons/fugue-icons/ui-panel-resize.png</iconset>
   </property>
   <property name="text">
    <string>Resi&amp;ze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMirror">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/layer-flip.png</normaloff>:/icons/fugue-icons/layer-flip.png</iconset>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="toolTip">
    <string>Mirror (M)</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/layer-flip-vertical.png</normaloff>:/icons/fugue-icons/layer-flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
   <property name="toolTip">
    <string>Flip (F)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionView_Detailed">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/magnifier.png</normaloff>:/icons/fugue-icons/magnifier.png</iconset>
   </property>
   <property name="text">
    <string>View &amp;Detailed</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionView_Animated">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/film.png</normaloff>:/icons/fugue-icons/film.png</iconset>
   </property>
   <property name="text">
    <string>View &amp;Animated</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHide_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionMovable_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionHide_statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Statusbar</string>
   </property>
  </action>
  <action name="actionMovable_statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Movable statusbar</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/arrow-turn.png</normaloff>:/icons/fugue-icons/arrow-turn.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHide_sidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sidebar</string>
   </property>
  </action>
  <action name="actionMovable_sidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sidebar</string>
   </property>
  </action>
  <action name="action_Ascending">
   <property name="text">
    <string>&amp;Ascending</string>
   </property>
  </action>
  <action name="action_Descending">
   <property name="text">
    <string>&amp;Descending</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/magnifier--plus.png</normaloff>:/icons/fugue-icons/magnifier--plus.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In (Ctrl++)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/magnifier--minus.png</normaloff>:/icons/fugue-icons/magnifier--minus.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out (Ctrl+-)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionpath_to_file">
   <property name="text">
    <string>path/to/file</string>
   </property>
  </action>
  <action name="actionasd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionasdsada">
   <property name="text">
    <string>asdsada</string>
   </property>
  </action>
  <action name="actionsdsd">
   <property name="text">
    <string>sdsd</string>
   </property>
  </action>
  <action name="actionasd_2">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionasdddd">
   <property name="text">
    <string>asdddd</string>
   </property>
  </action>
  <action name="actionDefullbright">
   <property name="text">
    <string>De-&amp;fullbright</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave_Selections_As">
   <property name="text">
    <string>Save Selections As...</string>
   </property>
  </action>
  <action name="actionExport_Images">
   <property name="text">
    <string>Export Images</string>
   </property>
  </action>
  <action name="actionExport_Selected_Images">
   <property name="text">
    <string>Export Selected As Images</string>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/arrow-circle-225-left.png</normaloff>:/icons/fugue-icons/arrow-circle-225-left.png</iconset>
   </property>
   <property name="text">
    <string>Rotate &amp;Left</string>
   </property>
   <property name="toolTip">
    <string>Rotate Left</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="icon">
    <iconset resource="fugue_icons.qrc">
     <normaloff>:/icons/fugue-icons/arrow-circle-315.png</normaloff>:/icons/fugue-icons/arrow-circle-315.png</iconset>
   </property>
   <property name="text">
    <string>Rotate &amp;Right</string>
   </property>
   <property name="toolTip">
    <string>Rotate Right</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionasd_3">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="fugue_icons.qrc"/>
 </resources>
 <connections/>
</ui>
